bomorinid/wavepool
# Allow only one cover story

This PR adds a custom clean method on the newspost change form so that only one newspost can be saved as cover story at a time.

Testing steps:
Go to a newspost in the CMS and set it as the cover story.
Go to a different newspost in the CMS and now set it as the cover story.
Go back to the first newspost and verify that it is no longer the cover story.
